third_party/spring_animation/README.md

```diff
@@ -0,0 +1,5 @@
+# spring_animation
```

Comment:
It's going to look like this dependency is unused. Should the README contain a warning against removing it?

Reply:
We'll be using it shortly.